app/controllers/profiles_controller.rb

```diff
@@ -40,6 +41,12 @@ def profile_params
     params.require(:profile).permit(:job, :bio, :linkedin, :github)
   end
 
+  def verify_user
+    mentee = current_user != User.find(Profile.find(params[:id]).user.id)
```

Comment:
I'll change this to `mentee = current_user != User.find(Profile.find(params[:id]).id)` after, don't want to keep committing.